create-vms.sh

```diff
@@ -18,6 +25,10 @@ fi
 erb ssh_public_key="`cat farm/.ssh/id_rsa.pub`" cloud-init/userdata.txt.erb > userdata.txt
 echo "instance-id: $(uuidgen)" > metadata.txt
 cloud-localds sonicpi-userdata.img userdata.txt metadata.txt
+virsh vol-create-as default sonicpi-userdata.img 0
+if virsh vol-list default | grep sonicpi; then
+  virsh vol-deletedefault sonicpi-userdata.img
```

Comment:
At a surface level, the vol-delete command doesn't accept pool then volume in the same order as vol-create-as. We can either just swap the order, volume pool, or explicitly call out the pool via `--pool`:

`virsh vol-delete sonicpi-userdata.img --pool ${LIBVIRT_POOL_NAME}`

And on a deeper reading, what are we going for with 28-31?  Looks like we try to create the volume, then if we don't find any volumes that match sonicpi (which should never be the case since we just created one), we try to delete sonicpi-userdata.img.  

I'm wondering if the intent was instead to see if the volume sonicpi-userdata.img already exists, if not create it.  (As is, the vol-create-as command will fail when run a second time.)